heliopy/data/test/test_data.py

```diff
@@ -165,14 +165,15 @@ def test_mag320ms(self):
         check_datetime_index(df)
 
     def test_mag15s(self):
-        df = imp.mag15s(self.probe, self.starttime, self.endtime)

```

Comment:
This shouldn't make a difference, since `self.probe` is `'8'` anyway I think.

Reply:
Oh yep, so sorry, forgot to remove this!